go.mod

```diff
@@ -19,14 +19,14 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/websocket v1.4.2
 	github.com/json-iterator/go v1.1.12
-	github.com/lucas-clemente/quic-go v0.28.1
 	github.com/mattn/go-colorable v0.1.13
 	github.com/miekg/dns v1.1.50
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.13.0
 	github.com/prometheus/client_model v0.2.0
-	github.com/rs/zerolog v1.20.0
+	github.com/quic-go/quic-go v0.33.0
```

Comment:
can quic-go be bumped to v0.34.0 ? looks like it has a decent amount of fixes compared to 0.33.0